src/foam/nanos/cron/Schedulable.js

```diff
@@ -43,11 +43,18 @@
       value: 'schedulableEventDAO'
     },
     {
-      name: 'schedule',
+      class: 'FObjectProperty',
       of: 'foam.nanos.cron.Schedule',
-      view: {
-        class: 'foam.u2.view.FObjectView',
-        of: 'foam.nanos.cron.SimpleIntervalSchedule'
+      name: 'schedule',
+      label: '',
+      factory: function(){
+        return foam.nanos.cron.SimpleIntervalSchedule.create();
+      },
+      view: function (_, X){
+        return {
+          class: 'foam.u2.view.SimpleIntervalScheduleView',
```

Comment:
This should be in the same package 
foam.nanos.cron. 
I'm not sure why package foam.u2.view keeps being used.

Reply:
Changed.